### DISCUSSION TITLE
REI not showing mod recipes and can't cheat any mod items, only vanilla ones

### DISCUSSION THREAD

**Discussion by jamesfarrington1** [shedaniel/RoughlyEnoughItems#1200] [Q & A] [answered]
When I hit r on an item from a mod, nothing happens. When I click on it to try and cheat myself that item, it says Error: Unknown item name: blah blah
This happens for all of the other mods I have
Is there an REI Plugin Compatibilities for fabric? I only see forge versions

Edit: Issue only occurs on server, not in singleplayer world

Fabric REI 1.19.2

**Comment by shedaniel** [comment, marked as answer]
I believe your server is using an un-standard /give command, I am unable to provide support for that
